chsdi/lib/translator.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+import os
+import psycopg2
+from cachetools import TTLCache
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import register_type, UNICODE
+
+CACHE_TTL = 3600  # Cache duration in seconds --> 1 hour cache for translations
+CACHE_SIZE = 10000
+
+
+# This is a class with a singleton approach. We do not need to have twenty caches around the code
+class Translator:
+
+    _translations = None
+    _supported_languages = ['en', 'fr', 'de', 'rm', 'it']
+
+    """
+    Database transation methods
+    """
+    @staticmethod
+    def get_db_connection():
+        return psycopg2.connect(host=os.environ.get('DBHOST'), dbname=os.environ.get('DBSTAGING'), user='www-data')
```

Comment:
thank you but the db name is missing now:
```
'bod_{}'.format(os.environ.get('DBSTAGING'))
```

Reply:
ha yeah, didn't think of that. I correct that